central/graphql/resolvers/image_vulnerabilities.go

```diff
@@ -263,9 +263,15 @@ func imageCveToVulnerabilityWithSeverity(in []*storage.ImageCVE) []Vulnerability
 }
 
 func (resolver *imageCVEResolver) withImageVulnerabilityScope(ctx context.Context) context.Context {
```

Comment:
This function should only be used in the postgres code, all of the legacy datastore usage is via conjunction queries I'm pretty sure. I'm not sure this change does anything